Composer/packages/client/src/pages/design/index.tsx

```diff
@@ -365,18 +384,38 @@ function DesignPage(props) {
           dialogId={dialogId}
           selected={selected}
           onSelect={handleSelect}
-          onAdd={() => actions.createDialogBegin({}, onCreateDialogComplete)}
           onDeleteDialog={handleDeleteDialog}
           onDeleteTrigger={handleDeleteTrigger}
-          openNewTriggerModal={openNewTriggerModal}
         />
         <div css={contentWrapper}>
-          {match && <ToolBar toolbarItems={toolbarItems} actions={actions} projectId={projectId} />}
+          {match && (
+            <ToolBar
+              toolbarItems={toolbarItems}
+              actions={actions}
+              projectId={projectId}
+              currentDialog={currentDialog}
+              openNewTriggerModal={openNewTriggerModal}
+              onCreateDialogComplete={onCreateDialogComplete}
+              onboardingAddCoachMarkRef={onboardingAddCoachMarkRef}
+            />
+          )}
           <Conversation css={editorContainer}>
             <div css={editorWrapper}>
               <div css={visualPanel}>
                 {breadcrumbItems}
-                <VisualEditor openNewTriggerModal={openNewTriggerModal} />
+                {dialogJsonVisible ? (
+                  <JsonEditor
+                    key={'testkey'}
+                    id={'modaljsonview'}
```

Comment:
nit: this id doesn't make sense any more since its not being rendering in the modal.